tqp/topic_queue_poller.py

```diff
@@ -17,17 +17,23 @@ def _jsonify_dictionary(dictionary):
 # -----------------------------------------------------------------------------
 
 
-def create_queue(queue_name, **kwargs):
+def create_queue(queue_name, tags=None, **kwargs):
+    tags = tags or {}
```

Comment:
```suggestion
def create_queue(queue_name, *, tags, **kwargs):
```

only caller always passes in a dict explicitly, no?

Reply:
sure